Fix the numpy version restriction, and eliminate the cmake and ninja
dependencies for which Void offers no Python package.

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -3,12 +3,9 @@
 requires = [
         "setuptools>=42.0.0",
         "wheel>=0.34.2",
-        "numpy;python_version >= '3.9' and platform_python_implementation == 'PyPy'",
-        "oldest-supported-numpy;python_version < '3.9' or platform_python_implementation != 'PyPy'",
+        "numpy",
         "scikit-build",
-        "cmake>=3.17",
         "nanobind>=1.9.2",
-        "ninja; platform_system!='Windows'",
         ]
 build-backend = "setuptools.build_meta"
 
